deps/deps.go

```diff
@@ -163,13 +163,19 @@ var dependencies = map[string]Dependency{
 
 // InstallDependency installs a binary to binDir, if ver is nil then the default version is used
 func InstallDependency(name, ver string, binDir string) error {
-	bin := fmt.Sprintf("%s/%s", binDir, name)
+	dependency, ok := dependencies[name]
+	var bin string
+	if dependency.BinaryName != "" {
```

Comment:
This line will cause a nil pointer exception if dependency is nil and not found.